Modules/Segmentation/Watersheds/include/itkWatershedSegmenter.hxx

```diff
@@ -176,9 +176,17 @@ Segmenter<TInputImage>::GenerateData()
   Self::MinMax(input, regionToProcess, minimum, maximum);
   // cap the maximum in the image so that we can always define a pixel
   // value that is one greater than the maximum value in the image.
-  if (NumericTraits<InputPixelType>::IsInteger CLANG_PRAGMA_PUSH CLANG_SUPPRESS_Wfloat_equal &&
+  if (NumericTraits<InputPixelType>::IsInteger
+// clang-format on
+CLANG_PRAGMA_PUSH
```

Comment:
`ITK_PRAGMA_PUSH` is normally followed by another macro, whereas `ITK_PRAGMA_POP` is usually alone.

Reply:
True.  What are you proposing?

Reply:
I thought that `// clang-format on` might not be needed for this case, but now I see that the macro gets put on the same line with some ordinary code.